config/application.rb

```diff
@@ -28,6 +28,7 @@ class Application < Rails::Application
     config.autoload_paths << "#{config.root}/app"
     config.autoload_paths << "#{config.root}/lib"
     config.autoload_paths += Dir["#{config.root}/lib/**/"]
+    config.autoload_paths << "#{config.root}/db/migrate"
```

Comment:
I think it'd be better to explicitly require that helper in the migration that needs it. They shouldn't ever be needed in production code outside of migrations.